dom-events/buttons.js

```diff
@@ -0,0 +1,25 @@
+function handleClick(event) {
+  console.log('button clicked');
+  console.log('event:', event);
+  console.log(event.target);
+}
+
+const clickButton = document.querySelector('.click-button');
+clickButton.addEventListener('click', handleClick);
```

Comment:
Don't forget to include that `$` in front of your variable names! Whenever we store DOM elements to a variable, we want to include that dollar sign

Reply:
Okay noted, will make sure to remember next time.